proposal/mpl_cartopy.tex

```diff
@@ -587,6 +587,20 @@ \subsubsection{General Maintenance}
 
 \subsubsection{Cartopy Refactor}
 
+\subsubsection{Consultatation Services}
+
+We will offer dedicated time to consult with NASA funded missions on
+usage of Matplotlib and Cartopy. This serves two primary purposes:
+immediate assistance to NASA funded missions and highlighting areas
+for improvements in documentation. This will include advice on how to use
+new developments, including the data refactor from the previous ROSES award
+and the new development work resulting from this award. Allocated specifically
```

Comment:
I would put a sentence or two in here saying something to indicate the direct benefit to the missions with an explicit example. Here is a quick go at what I mean:

NASA Missions will be able to bring their problems and ideas directly to the Matplotlib team to get a tailored solution for their problem. Rather than mission scientists spending 10-20 hours figuring out how to do something, the Matplotlib team could help them with in just an hour or two due to the Matplotlib team's expertise in their tools. This will end up saving missions time and money.